app/code/Magento/Customer/Block/Form/Register.php

```diff
@@ -223,4 +223,13 @@ public function getRequiredCharacterClassesNumber()
     {
         return $this->_scopeConfig->getValue(AccountManagement::XML_PATH_REQUIRED_CHARACTER_CLASSES_NUMBER);
     }
+
+    /**
+     * Get show address fields
+     *
+     * @return bool
+     */
+    public function getShowAddressFields():bool {
```

Comment:
Unfortunately, we are not allowed to add new public methods to @api classes according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)